cli/README.adoc

```diff
@@ -42,14 +42,16 @@ pulled/pushed.
 
 == Rotate Root Keys
 
+=== Rotate Repo Server Root Keys
+
 1. Initialize a local ota-plus cache using credentials.zip downloaded
 from ATS Garage:
 
-    garage-sign init --repo myrepo --credentials credentials.zip --reposerver <reposerver uri>
+    garage-sign init --repo myrepo --servertype reposerver --credentials credentials.zip --reposerver <reposerver uri>
```

Comment:
Can we have something like:

`garage-sign repo init ....`
`garage-sign director init ...` ?

Reply:
How about `garage-sign init director` and `garage-sign init repo` since there are not other subcommands of `repo` and `director`?

Reply:
Ok, please ignore my comment, I thought there were more commands.